### PR TITLE
Private Marketplace docs improvements

### PR DESCRIPTION
**Pull request by albertotessarotto**
## Pull Request Type

- [x] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [ ] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [ ] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [ ] No sensible content has been committed

### CODE REVIEW

File: docs/marketplace/add_to_marketplace/contributing_overview.md [hunk 1]
```diff
@@ -10,15 +10,15 @@ In this section, we dive into the process of creating a new component for the Ma
 Marketplace items can only be created and managed by people with access to the Mia-Platform Console CMS, therefore you may have to ask your Mia-Platform referent to add your Marketplace items for you.
 :::
 
-## Public and private marketplace
+## Public and private Marketplace
 
-The Mia-Platform Marketplace is the place where every Company can find most of the components needed to set up its architecture. It is publicly available on the Console, meaning that all Companies present on the platform can view and make use of its components.
+Mia-Platform Marketplace is the place where every Company can find most of the components needed to set up its project architecture. It is publicly available on Mia-Platform Console, meaning that all Companies can view and make use of its items.
```
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/contributing_overview.md:15]:
```suggestion
Mia-Platform Marketplace is the place where every Company can find most of the components needed to set up its Project architecture. It is publicly available on Mia-Platform Console, meaning that all Companies can view and make use of its items.
```

**Review by fredmaggiowski — APPROVED** [review]

File: docs/marketplace/add_to_marketplace/contributing_overview.md [hunk 2]
```diff
@@ -10,15 +10,15 @@ In this section, we dive into the process of creating a new component for the Ma
...
 
 However, a Company might want to have its own private Marketplace, with private components that must not be shared with everyone else on the Console. 
 
 To achieve this, a Company Owner or a Project Administrator can create new private components **from the CMS** by specifying their Company's identifier in the `tenantId` field.
 
-Company Owners and Project Administrators can create, delete and update Company-wide markeplace elements, but they cannot manage public Marketplace items.  
+Company Owners and Project Administrators can create, delete and update those Markeplace items exclusively available for their Company, but they cannot manage public Marketplace items.  
 The Console Super User role, instead, has permission to manage all Marketplace items from the CMS without any limitations.
 
 If you want to contribute to the **public Mia-Platform Marketplace** with a new component instead, and make it available for everyone, you can [open an issue](https://github.com/mia-platform/community/issues/new?labels=marketplace&template=marketplace-contribution.md&title=Add+new+marketplace+item) on GitHub, wherein you will find what information is needed for the request.
```
[the remark this answers is not in the document]
Comment (fredmaggiowski) [reply docs/marketplace/add_to_marketplace/contributing_overview.md:24]: the link opens the issue submission page, not an empty issue, what do you mean with "the submitting template"?

[the remark this answers is not in the document]
Comment (fredmaggiowski) [reply docs/marketplace/add_to_marketplace/contributing_overview.md:24]: oh the link has changed

```suggestion
If you want to contribute to the **public Mia-Platform Marketplace** with a new component instead, and make it available for everyone, you can [open an issue](https://github.com/mia-platform/community/issues/new?assignees=%40mia-platform%2Fsig-marketplace&labels=marketplace&projects=&template=marketplace-contribution.yaml&title=%5BNew+marketplace+item%5D%3A+) on GitHub, wherein you will find what information is needed for the request.
```

**Review by fredmaggiowski — APPROVED** [review]

File: docs/marketplace/add_to_marketplace/contributing_overview.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (marcofilippi) [reply docs/marketplace/add_to_marketplace/contributing_overview.md]: Exactly this one!

**Pull request merged**